src/core/database/database.service.ts

```diff
@@ -572,30 +572,29 @@ export class DatabaseService {
     baseNodeLabel: string;
     aclEditProp?: string;
   }): Promise<void> {
-    const aclEditPropName =
-      aclEditProp || `canEdit${upperFirst(baseNodeLabel)}`;
     await this.createBaseNode<TObject>({
       session,
       baseNodeLabel,
       input,
       acls,
-      aclEditProp: aclEditPropName,
+      aclEditProp,
     });
-    await Promise.all(
-      Object.keys(input)
-        .filter(key => !(key === 'id' || key === 'userId'))
-        .map(async key => {
-          await this.createProperty({
-            session,
-            key,
-            value: input[key as keyof TObject] as string,
-            id: input.id!,
-          });
-        })
-    );
+
+    for (const k in input) {
+      if (k === 'id' || k === 'userId') {
+        continue;
+      }
```

Comment:
Shouldn't these just not be passed in? Or maybe `id` is special. But def not `userId` other relationships will be named differently (`languageId`, `projectId`, etc.)

Reply:
I left `id` in there and eliminated all instances where `userId` was being passed in 👍